src/store/actions/performNextAction/send.js

```diff
@@ -12,6 +14,17 @@ async function waitForConfirmations ({ getters, dispatch }, { transaction, netwo
   )
   try {
     const tx = await client.chain.getTransactionByHash(transaction.txHash)
+
+    // In case transaction doesn't exist: wait for 30 minutes then return failed state
+    // Could happen on Dropped&Replaced
+    if (!tx && hasTimedOut(transaction)) {
+      return {
+        endTime: Date.now(),
+        status: TxStatus.Failed
```

Comment:
TxStatus.Failed. - what is the value of this. 
Is it one of the supported states - FAILED/ REFUND etc

Reply:
Values is 'FAILED'